community-plugins.json

```diff
@@ -6788,5 +6788,12 @@
     "author": "Maciej Jur",
     "description": "Basic support for Markdoc files.",
     "repo": "kamoshi/obsidian-markdoc"
+ },
+ {
+    "id": "chem",
+    "name": "Chem",
+    "author": "Acylation",
+    "description": "Chemistry support for Obsidian",
```

Comment:
We have recently published a guide for plugin descriptions to follow here: https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Keep+plugin+descriptions+short+and+simple

It's a bit redundant to say "for Obsidian" so I suggest you instead give a clearer idea of what this plugin does (specifically rendering chemical structures.

Reply:
Thanks for the kind review. Changes have been made both here and in the plugin repo as required, and I have release a new version with an updated manifest. 

The new descriptions is "Providing chemistry supports. Rendering SMILES strings into chemistry structures."